mlflow/pyfunc/__init__.py

```diff
@@ -409,23 +412,34 @@ def predict(self, data: PyFuncInput) -> PyFuncOutput:
                      (i.e. read / write the elements using C-like index order), and DataFrame
                      column values will be cast as the required tensor spec type.
 
+        :param parameters: Additional parameters to pass to the model for inference.
+
         :return: Model predictions as one of pandas.DataFrame, pandas.Series, numpy.ndarray or list.
         """
         input_schema = self.metadata.get_input_schema()
         if input_schema is not None:
             data = _enforce_schema(data, input_schema)
 
+        if parameters is not None:
+            if not isinstance(parameters, dict):
```

Comment:
This will be handled by `_enforce_parameters_schema`, right? I think we can remove this here.